<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialPropertiesTab</class>
 <widget class="QWidget" name="MaterialPropertiesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="tabToggle">
     <property name="text">
      <string>Material</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ambient:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ambientColorRoot"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Diffuse:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="diffuseColorRoot"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Specular:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="specularColorRoot"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Shininess:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
          <item>
           <widget class="QDoubleSpinBox" name="shininessValue">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>128.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="shininessSlider">
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Effect:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="effectFileRoot" stretch=""/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="textureSelectorRoot"/>
        </item>
        <item>
         <widget class="QPushButton" name="addTextureSelector">
          <property name="text">
           <string>Add Texture Slot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
